qmlti/lib.php

```diff
@@ -250,13 +263,14 @@ function init_db($db) {
 
         $sql = <<< EOD
 CREATE TABLE [dbo].[{$results_table_name}] (
+  [consumer_key] VARCHAR(50) NOT NULL DEFAULT '',
   [context_id] VARCHAR(255),
   [assessment_id] VARCHAR(255),
   [customer_id] VARCHAR(25),
   [created] DATETIME,
   [score] VARCHAR(255),
-  [result_id] VARCHAR(255),
- CONSTRAINT [PK_{$results_table_name}] PRIMARY KEY CLUSTERED ([context_id] ASC, [assessment_id] ASC, [result_id] ASC)
+  [result_id] INT,
+ CONSTRAINT [PK_{$results_table_name}] PRIMARY KEY CLUSTERED ([consumer_key] ASC, [result_id] ASC)
```

Comment:
Not 100% sure I understand the purpose of this table but the PK feels wrong (would the PK not be consumer_key, context, resourceid (or assessmentid in QM) and userid?